src/views/Material.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <v-app-bar
+      :clipped-left="$vuetify.breakpoint.lgAndUp"
+      app
+      color="blue-grey darken-3"
+      dark
+    >
+      <v-avatar :tile="true" width="140px">
+        <img :src="require('@/assets/logo.svg')" alt="mova Logo" />
+      </v-avatar>
+      <v-avatar :tile="true">
+        <img :src="require('@/assets/logomat-logo.svg')" alt="LOGomat Logo" />
+      </v-avatar>
+      <v-toolbar-title style="width: 300px" class="ml-0 pl-4">
+        <span class="hidden-sm-and-down">LOGomat</span>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-btn icon title="Logomat" to="/">
+        <v-icon>mdi-arrow-left</v-icon>
+      </v-btn>
+      <Settings />
```

Comment:
Could factor this out as appbar-shared-right or similar. Same for Order component.